HHS/Head-Start-TTADP
# [TTAHUB-1312] When a goal is placed in "edit mode", the side navigation in the activity report should revert to "in progress."

## Description of change
Once a goal is saved on an activity report, re-opening that goal for editing can cause the side navigation to become hopelessly out of sync with the actual state of the page. This PR addresses this bug.

## How to test
Save a goal. Edit that goal. Paginate through the activity report. As you do so, keep in mind that if the goal is open for editing, the side nav should say "in progress" and if the goal is saved and in the review state, it should say "complete". You should not be able to submit a report with the goal in an open state.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1312


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
